evaluation/src/main/java/com/formulasearchengine/mlp/evaluation/GoldEntry.java

```diff
@@ -1,12 +1,13 @@
 package com.formulasearchengine.mlp.evaluation;
 
+import java.io.Serializable;
 import java.util.List;
 import java.util.Set;
 
 /**
  * Created by Leo on 05.10.2016.
  */
-public class GoldEntry {
+public class GoldEntry implements Serializable {
```

Comment:
I cant see how to do this without rewriting pushed history, therefore I have done it in a new pull-request #128